xtas/tasks/single.py

```diff
@@ -282,7 +282,7 @@ def sentiwords_tag(doc, output="bag"):
         return counts
 
     elif output == "tokens":
-        return [ngram if polarity == 0 else (ngram, polarity)
```

Comment:
Would it make sense to return (ngram, None) here if there's no polarity information, so the return type is more regular? That seems easier to use, although I can also see this work with isinstance().